js/examples/scrubbing/index.html

```diff
@@ -15,7 +15,7 @@ <h3>scroll to scrub!</h3>
 
         </main>
 
-        <script src="../../../dist/rive.min.js"></script>
+        <script src="/dist/rive.min.js"></script>
```

Comment:
I moved the example out of parcel to here; the frameworks directory is mostly for testing parcel/Vue/React. The root folder is meant to hold examples of how to use the api.